src/java.base/share/classes/java/util/zip/Inflater.java

```diff
@@ -49,23 +49,24 @@
  * thrown.
  * <p>
  * This class inflates sequences of ZLIB compressed bytes. The input byte
- * sequence is provided in either byte array or byte buffer, via one of the
+ * sequence is provided in either byte array or {@link ByteBuffer}, via one of the
  * {@code setInput()} methods. The output byte sequence is written to the
- * output byte array or byte buffer passed to the {@code inflate()} methods.
+ * output byte array or {@code ByteBuffer} passed to the {@code inflate()} methods.
  * <p>
- * The following code fragment demonstrates a trivial compression
- * and decompression of a string using {@code Deflater} and
- * {@code Inflater}.
- * {@snippet id="compdecomp" lang="java" class="Snippets" region="DeflaterInflaterExample"}
+ * To release resources used by the {@code Inflater}, applications must call the
+ * {@link #end()} method. After {@code end()} has been called, subsequent calls
+ * to several methods of the {@code Inflater} will throw an {@link IllegalStateException}.
```

Comment:
Since `close()` is not mentioned here, there may be some come confusion about whether `end()` still needs to be called withing T-W-R.